src/fswAlgorithms/imageProcessing/opticalFlow/opticalFlow.h

```diff
@@ -62,7 +62,8 @@ class OpticalFlow: public SysModel {
     std::vector<cv::Vec2f> firstFeatures;
 
 public:
-    std::string filename = "";  //!< Filename for module to read an image directly
+    std::string directoryName = "";  //!< Directory name for module to read an image directly
+    std::string imageFilExtension = ".png";  //!< Directory name for module to read an image directly
```

Comment:
:large_orange_diamond: Did you forget the `e` in `imageFilExtension`?

Reply:
Fixed